source/lex.tex

```diff
@@ -296,13 +297,15 @@
 \ucode{003d} & \uname{equals sign} & \tcode{=} \\
 \ucode{003e} & \uname{greater-than sign} & \tcode{>} \\
 \ucode{003f} & \uname{question mark} & \tcode{?} \\
+\ucode{0040} & \uname{commercial at} & \tcode{@} \\
 \ucode{0041} .. \ucode{005a} & \uname{latin capital letter a .. z} & \tcode{A B C D E F G H I J K L M} \\
  & & \tcode{N O P Q R S T U V W X Y Z} \\
 \ucode{005b} & \uname{left square bracket} & \tcode{[} \\
 \ucode{005c} & \uname{reverse solidus} & \tcode{\textbackslash} \\
 \ucode{005d} & \uname{right square bracket} & \tcode{]} \\
 \ucode{005e} & \uname{circumflex accent} & \tcode{\caret} \\
 \ucode{005f} & \uname{low line} & \tcode{_} \\
+\ucode{0060} & \uname{grave accent} & \tcode{`} \\
```

Comment:
The result is very similar to apostrophe.
Is this the intended rendering?
![1687085211](https://github.com/cplusplus/draft/assets/21071787/59b02b83-0ba9-4d97-a13f-8c640e4c37a5)
![1687085217](https://github.com/cplusplus/draft/assets/21071787/96520687-3e73-41ec-8d2e-673dd55e17a5)

Reply:
That's because the grave sigil is a LaTeX macro for the opening single quote -- thanks for spotting!